src/components/NavBar.js

```diff
@@ -48,8 +49,16 @@ class NavBar extends Component {
           <Menu.Item as={NavLink} to="/signup" id="sign-up-button">Sign Up</Menu.Item>
         </>
       )
+    }
 
+    if (this.props.currentUser.isSignedIn === true) {
+      userProfileButton = (
+        <>
+          <Menu.Item as={NavLink} to="/profile" id="profile-button">Profile</Menu.Item>
+        </>
+      )
```

Comment:
Can all user roles see their profile or should this be re-written to only subscribed university (as on line 31-32)

Reply:
All users will access the profile component, then the profile component content will vary depending on the user role